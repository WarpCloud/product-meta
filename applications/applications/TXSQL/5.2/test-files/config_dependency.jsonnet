{
  application_type: "TXSQL",
  application_name: "txsql1",
  application_version: "5.2",
  user_config: {},
}
