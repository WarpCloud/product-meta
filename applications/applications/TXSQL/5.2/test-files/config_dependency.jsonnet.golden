{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "txsql1",
            "application_type": "TXSQL"
         },
         "configs": {
            "txsql_cpu_limit": 2,
            "txsql_cpu_request": 0.10000000000000001,
            "txsql_memory_limit": 4,
            "txsql_memory_request": 1
         },
         "instance_settings": { },
         "moduleName": "txsql",
         "name": "txsql1-txsql",
         "version": "5.2"
      }
   ],
   "instance_settings": { },
   "version": ""
}
